src/browser/ui/dom/DOMPropertyOperations.js

```diff
@@ -111,6 +126,22 @@ var DOMPropertyOperations = {
   },
 
   /**
+   * Creates markup for a custom property.
+   *
+   * @param {string} tagName
+   * @param {string} name
+   * @param {*} value
+   * @return {?string} Markup string, or null if the property was invalid.
+   */
+  createMarkupForCustomAttribute: function(name, value) {
```

Comment:
Would it make sense to just combine this into createMarkupForProperty (and change the signature there)